python/lsst/ts/watcher/rules/dew_point_depression.py

```diff
@@ -138,34 +160,35 @@ def get_schema(cls):
               - sensor_names
             additionalProperties: false
       required:
-        - sal_name
+        - sal_index
         - topics
       additionalProperties: false
-    required:
-      - index
-      - topics
-    additionalProperties: false
   temperature_sensors:
-    description: Temperature sensors to monitor.
+    description: >-
+        ESS temperature sensors to monitor.
+        These can be any topic with a "temperature" field,
+        including temperature, hx85a and hx85ba.
     type: array
     minItems: 1
     items:
       type: object
       properties:
-        sal_name:
-          description: 'SAL component name[:SAL index] e.g. ESS:1'
-          type: string
+        sal_index:
+          description: SAL index of ESS CSC
```

Comment:
Please make sure the description ends in a period.

Reply:
I have gone through all descriptions for the three rules I worked on in this ticket and made sure they all end with period.